Ginger/GingerCoreNETUnitTest/TestResources/Repository/Flow 1.Ginger.BusinessFlow.xml

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<GingerRepositoryItem><Header ItemGuid="cf613b63-26ae-47e0-8d3a-0808163e0dce" ItemType="BusinessFlow" CreatedBy="pravishe" Created="202209160601" GingerVersion="4.5.0.0" Version="9" LastUpdateBy="pravishe" LastUpdate="202209210524" />
```

Comment:
Rename the xml to have more meaningful name

Reply:
Done